### DISCUSSION TITLE
[Feature Request] Granular Permissions for Rules Namespaces/Groups

### DISCUSSION THREAD

**Discussion by tonychoe** [grafana/grafana#40362] [Alerting]
Can we set granular permissions for Cortex/Loki-managed alerting/recording rules namespaces and groups? This would be particularly useful for the large org/company.

The implementation may follow the permissions design for [dashboards and dashboard folders](https://grafana.com/docs/grafana/latest/permissions/dashboard-folder-permissions/).

For examples, the permission levels per organization roles:
- Admin - Can create, edit, or delete namespaces and groups. Can also change namespaces and groups permissions.
- Edit - Can create and edit groups. _Cannot_ change namespaces or groups permissions, or add, edit, or delete namespaces.
- View - Can only view existing namespaces and groups.

For example, when manage permissions of namespace, the namespace permissions apply to the namespace and all groups contained within it.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
